builder/policy/policy.spec.ts

```diff
@@ -5,36 +5,44 @@ it('should neverBuiltDependencies', () => {
   const allowBuild = createAllowBuildFunction({
     neverBuiltDependencies: ['foo'],
   })
-  expect(allowBuild('foo')).toBeFalsy()
-  expect(allowBuild('bar')).toBeTruthy()
+  if (allowBuild) {
+    expect(allowBuild('foo')).toBeFalsy()
+    expect(allowBuild('bar')).toBeTruthy()
+  }
```

Comment:
is there another way to fix this? this way the test will pass if allowBuild is not defined.

at least like this 
```suggestion
  expect(typeof allowBuild).toBe('function')
  if (allowBuild) {
    expect(allowBuild('foo')).toBeFalsy()
    expect(allowBuild('bar')).toBeTruthy()
  }
```

Reply:
Ok, I'll apply this